sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -1,16 +1,9 @@
 # Release History
 
-## 1.1.0b2 (Unreleased)
 
-### Features Added
-- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
```

Comment:
Why did you remove the release notes?

Reply:
MicrosoftTeamsAppIdentifier feature has already been released. But not aware they added release note to rooms changeslog. In this case, I think we can keep the Features Added note and just replace the ## 1.1.0b2 (Unreleased) with out release version and description.

Reply:
Updated

Reply:
I removed the release notes from the communication identifier as they seem to be unreleased and cause the conflicts. The change in the changelog file was introduced here https://github.com/Azure/azure-sdk-for-python/commit/33608ac52f65386d0ba923446e8664c41e32c2a1#diff-998a1fc3d0f56f4f5e35b2e6099c151bffc6db6fdba8cbf411db895525d3ba40 but is not needed in rooms changeLog